meshtastic/interfaces/ble/notifications.py

```
@@ -228,3 +265,693 @@ def _get_callback(self, characteristic: str) -> Callable[[Any, Any], None] | Non
+            logger.debug(
+                "safe_execute callable-only probe failed for notification handler (%s): %s.",
+                error_msg,
+                exc,
+                exc_info=True,
+            )
+            if executed:
+                _report_handler_failure()
+                return
+            _fallback_if_not_executed()
+
+    def from_num_handler(self, _: Any, b: bytes | bytearray) -> None:
```

Ruff (0.15.6)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `_`

(ANN401)